Arrangement_on_surface_2/test/Arrangement_on_surface_2/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ enable_testing()
 
 find_package(CGAL REQUIRED COMPONENTS Core)
 
+include (${CGAL_USE_FILE})
```

Comment:
Have you found out why that is necessary?

Reply:
No I gave up. As I wrote in the commit message, I suspect it has to do with the `flags` variable that is not passed to cmake if the use-file is not included (maybe a scope issue?). Note that it is an issue for non cmake testsuite only. As it was in a test and not an example I considered that it could be fixed later (arrangement is a particular case anyway)